packages/vats/src/ibc.js

```diff
@@ -454,59 +426,22 @@ export const prepareIBCProtocol = (zone, { makeVowKit, watch, when }) => {
               //     e => console.warn('Manual packet', e, 'failed:', e),
               //   );
 
-              const attempt = await when(
+              return watch(
                 /** @type {ProtocolImpl} */ (protocolImpl).inbound(
                   localAddr,
                   remoteAddr,
                 ),
+                this.facets.protocolImplInboundWatcher,
+                {
+                  channelKeyToAttempt,
+                  channelKeyToInfo,
+                  obj,
+                  util,
```

Comment:
These can be obtained from `this.state` and `this.facets.util`.
```suggestion
                  obj,
```